bats/tests/helpers/defaults.bash

```diff
@@ -201,6 +195,14 @@ using_dev_mode() {
 }
 
 ########################################################################
+# Kubernetes versions
+
+# The main Kubernetes version to test.
+: "${RD_KUBERNETES_VERSION:=1.23.6}"
+
+# A secondary Kubernetes version; this is used for testing upgrades.
+: "${RD_KUBERNETES_ALT_VERSION:=1.22.7}"
```

Comment:
You swapped the default and alternate versions in this change. Was this intentional?

I don't think it matters either way, but if we want to test with a newer version by default, maybe we should move even higher than 1.23 and to to 1.27 or so instead.

Reply:
Yes, because even going by the variable names we can see that we _meant_ to use the newer version most of the time, but we had a buggy implementation that defaulted to the version to upgrade from instead.  It made no sense to default to the version that had extra qualifiers on it.

Reply:
> going by the variable names we can see that we _meant_ to use the newer version most of the time

That's actually not true. The names are a historical accident, and I always felt bad about the fact that the default version was named `prev` just because of the upgrade test. It is all my fault; I just never got around to renaming it.

I don't actually care that you swapped them, I just wanted to understand if there was a deeper reason behind it.

Reply:
Ah, in that case, the reason is just because you made it confusing :p